server/index.js

```diff
@@ -50,9 +51,26 @@ app.post('/api/test', (req, res) => {
   TastyRunner.run(filters.type)
     .then((stats) => {
       io.emit('tests:end', stats);
+
+      config.get('notification').map(notification => {
+        if (notification.type) {
+          io.emit('tests:test', notification.type);
+          const { isNotificationEnabled, notify, getMessage } = require(`./notification/${notification.type}`);
```

Comment:
@DariaLutkova What should happen, if we have type in our configuration file, but don't have a module with implementation?
If I add notification service with type OpsGenie, I'll pass 56's line, but I will fail here

Reply:
It will fall with error like this: Cannot find module './notification/slack'